plugins/core-ui/src/main/resources/app/pentaho/util/url.js

```diff
@@ -93,7 +94,7 @@ define([
       protocol: protocol,
       username: auth.length > 0 ? auth[0] : "",
       password: auth.length > 1 ? auth[1] : "",
-      hostname: host,
+      hostname: xssUtil.sanitizeHtml(host),
```

Comment:
I don't understand what we are trying to fix here. The JIRA issue does not also list the App Scan findings.